apps/meteor/client/views/teams/contextualBar/channels/RoomActions.js

```diff
@@ -81,10 +82,24 @@ const RoomActions = ({ room, reload }) => {
 	const menuOptions = useMemo(() => {
 		const AutoJoinAction = async () => {
 			try {
-				await updateRoomEndpoint({
+				const { room: updatedRoom } = await updateRoomEndpoint({
 					roomId: rid,
 					isDefault: !room.teamDefault,
 				});
+
+				if (!room.teamDefault) {
```

Comment:
```suggestion
				if (updatedRoom.teamDefault) {
```
Maybe this way it's easier to read.